saleor/graphql/account/mutations/account.py

```diff
@@ -0,0 +1,209 @@
+import graphene
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+
+from ....account import emails, events as account_events, models, utils
+from ....checkout import AddressType
+from ...account.enums import AddressTypeEnum
+from ...account.types import Address, AddressInput, User
+from ...core.mutations import BaseMutation, ModelMutation
+from .base import BaseAddressDelete, BaseAddressUpdate, send_user_password_reset_email
+from .staff import CustomerCreate
+
+
+class AccountRegisterInput(graphene.InputObjectType):
+    email = graphene.String(
+        description="The unique email address of the user.", required=True
+    )
+    password = graphene.String(description="Password", required=True)
+
+
+class AccountRegister(ModelMutation):
+    class Arguments:
+        input = AccountRegisterInput(
+            description="Fields required to create a user.", required=True
+        )
+
+    class Meta:
+        description = "Register a new user."
+        exclude = ["password"]
+        model = models.User
+
+    @classmethod
+    def save(cls, info, user, cleaned_input):
+        password = cleaned_input["password"]
+        user.set_password(password)
+        user.save()
+        account_events.customer_account_created_event(user=user)
+
+
+class AccountInput(graphene.InputObjectType):
+    first_name = graphene.String(description="Given name.")
+    last_name = graphene.String(description="Family name.")
+    default_billing_address = AddressInput(
+        description="Billing address of the customer."
+    )
+    default_shipping_address = AddressInput(
+        description="Shipping address of the customer."
+    )
+
+
+class AccountUpdate(CustomerCreate):
+    class Arguments:
+        input = AccountInput(
+            description="Fields required to update the account of the logged-in user.",
+            required=True,
+        )
+
+    class Meta:
+        description = "Updates the account of the logged-in user."
+        exclude = ["password"]
```

Comment:
Instead of excluding fields, safer is to explicitly pass a list of included fields

Reply:
As we spoke this excludes fields from validation, not from the output. @fowczarek I would go through all `exclude` that this PR adds and see if they're actually needed. They mostly try to exclude password but that should be already excluded at the `User` type level.